src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -575,8 +575,7 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
       <MoveItemsDialog
         items={[...items]}
         onItemsMoved={() => {
-          dispatch(fetchSortedFolderContentThunk(currentFolderId));
-          onItemsMoved?.();
+          !isTrash && dispatch(fetchSortedFolderContentThunk(currentFolderId));
```

Comment:
I would say that this does not affect trash items even if you are in trash, in fact, it is necessary to update Drive so that when the user comes back they will find the folder updated.